go/vt/topo/consultopo/server.go

```diff
@@ -21,29 +21,42 @@ package consultopo
 
 import (
 	"encoding/json"
-	"flag"
 	"fmt"
 	"os"
 	"strings"
 	"sync"
 	"time"
 
 	"github.com/hashicorp/consul/api"
-
-	"vitess.io/vitess/go/vt/vterrors"
+	"github.com/spf13/pflag"
 
 	"vitess.io/vitess/go/vt/log"
+	"vitess.io/vitess/go/vt/servenv"
 	"vitess.io/vitess/go/vt/topo"
+	"vitess.io/vitess/go/vt/vterrors"
 )
 
 var (
-	consulAuthClientStaticFile = flag.String("consul_auth_static_file", "", "JSON File to read the topos/tokens from.")
-	// serfHealth is the default check from consul
-	consulLockSessionChecks = flag.String("topo_consul_lock_session_checks", "serfHealth", "List of checks for consul session.")
-	consulLockSessionTTL    = flag.String("topo_consul_lock_session_ttl", "", "TTL for consul session.")
-	consulLockDelay         = flag.Duration("topo_consul_lock_delay", 15*time.Second, "LockDelay for consul session.")
+	consulAuthClientStaticFile string
+	consulLockSessionChecks    = "serfHealth"
+	consulLockSessionTTL       string
+	consulLockDelay            = 15 * time.Second
 )
 
+func init() {
+	for _, cmd := range []string{"vtbackup", "vtcombo", "vtctl", "vtctld", "vtgate", "vtgr", "vttablet", "vttestserver", "zk"} {
+		servenv.OnParseFor(cmd, registerServerFlags)
+	}
+}
+
+func registerServerFlags(fs *pflag.FlagSet) {
+	fs.StringVar(&consulAuthClientStaticFile, "consul_auth_static_file", consulAuthClientStaticFile, "JSON File to read the topos/tokens from.")
+	// serfHealth is the default check from consul
```

Comment:
this comment probably belongs with the default value (in the `var` block above) as it references "serfHealth" which doesn't appear on this line at all